packages/usa-in-page-navigation/src/usa-in-page-navigation.json

```diff
@@ -47,8 +47,18 @@
+    },
+    {
+      "id": 8,
+      "section_heading": "This header is hidden with display:none and should not appear in the in-page nav",
+      "is_subheading": false,
+      "hidden_style": "display: none"
+    },
+    {
+      "id": 9,
+      "section_heading": "This header is hidden with visibility: hidden and should not appear in the in-page nav",
+      "is_subheading": false,
+      "hidden_style": "visibility: hidden"
     }
```

Comment:
I'd remove these new items. This seems more like test code.

Reply:
Yep, this is outdated code. Good catch! Removed in https://github.com/uswds/uswds/pull/5393/commits/6916eb1eeda1900cae0039c0f788d13915413016